build-system/common/utils.js

```diff
@@ -145,6 +147,12 @@ function getFilesToCheck(globs, options = {}, ignoreFile = undefined) {
       log(green('INFO: ') + 'No files to check in this PR');
       return [];
     }
+    /** forbidden-terms always needs to be checked. */
+    if (
+      !filesChanged.includes('build-system/test-configs/forbidden-terms.js')
```

Comment:
nit: To avoid repetition, can we capture the filename as a `const forbiddenTerms =` and reuse it in the two `if` statements?